sdcm/nemesis.py

```diff
@@ -2818,16 +2818,15 @@ def execute_data_validation_thread(command_template, keyspace_name, number_of_ro
             self.log.info("Restoring the schema of the keyspace '%s'", chosen_snapshot_info["keyspace_name"])
             restore_task = mgr_cluster.create_restore_task(restore_schema=True, location_list=location_list,
                                                            snapshot_tag=chosen_snapshot_tag)
-            step = 10
-            extra_time = 300
-            restore_task.wait_and_get_final_status(step=step,
-                                                   timeout=chosen_snapshot_info['expected_timeout'] / step + extra_time)
+
+            restore_task.wait_and_get_final_status(step=10, timeout=chosen_snapshot_info['expected_timeout'])
```

Comment:
why can't we just revert this change ? to the static timeout it had ? 

this shouldn't take 30min and even not 15min, it's just the system tables 
if it's stuck we have a bug somewhere, and should be handled, not waiting more.

Reply:
because it is not being enough... and the case i'm currently testing (the most challenging one) is this nemesis during prepare (in the individual nemesis), so it won't be enough 120 seconds

Reply:
The expected timeouts are those measured for the actual restore aren't they ?

It's a bit weird to expect the restore of the scheme to take as long the actual data restore, maybe 2min isn't enough and we need to enlarge it by to 10min, but it should be based on the number that were defined for the data

Reply:
again, my suspicious is about these nemesis runs during prepare taking much much longer than initially planned